data.access/src/test/java/data/access/DatabaseUtility.java

```diff
@@ -0,0 +1,139 @@
+package data.access;
+import java.sql.*;
+import java.util.ArrayList;
+
+
+public class DatabaseUtility implements DatabaseAccessor {
+
+	static final String DB_URL = "jdbc:mysql://localhost:3306/sakila";
+	static final String USER = "root";
+	static final String PASS = "password";
+
+	public DataRow[] executeQuery(String sql) {
```

Comment:
Most of the method bodies are duplicates. Refactor to share code (create a private method)